visgraph/graphcore.py

```diff
@@ -38,9 +38,11 @@ class Graph:
         Edges are directional sets of a from node-id and to node-id and
         a piece of arbitrary edge information.
     '''
-    def __init__(self):
+    def __init__(self, merge=None):
```

Comment:
so, i think i'd be all for mainlining this if it weren't for the optional parameter in the constructors.  if one is calling the constructor, they could just as easily call g.merge() after construction.
It's probably nearly phillosophical/aesthetic but if every new thing we made it do was also expected to be possible directly from the ctor, it'd get pretty nasty.  Also, since it's not storing anything, seems to make less sense as a constructor arg.